### ISSUE TITLE
Staging Review finding: Textareas don't communicate hint text/limit

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#124718]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Benefits Intake Optimization (Aquia)
**Product:** Form 21-2680
**Feature:** MVP Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User can't perceive an element.](https://design.va.gov/about/experience-standards/#user-can-perceive-all-meaningful-elements)
**VA.gov Experience Standard - category:** [Comprehension](https://design.va.gov/about/experience-standards/#category-comprehension)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility)

### Description

Textareas throughout the experience have a hard maxlength, however this limit is not communicated and when the user hits this limit, there's no indication to screen reader users, although there is visual help text that appears for sighted users. 

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="684" height="1012" alt="character count" src="https://github.com/user-attachments/assets/bcf63f1e-afe9-4647-82fd-eeafd5e8c0bf" />

</details>

## Recommended action

Use the textarea with character count from VADS - https://design.va.gov/components/form/textarea#with-character-count

This component communicates the textarea limitation up front and provides feedback when a user goes over the limit. 

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 3.3.2 Labels or Instructions - Level A](https://www.w3.org/TR/WCAG22/#labels-or-instructions)
- **Modality:** Screen reader
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://dj540s05.optimalworkshop.com/questions/f54355271969f160ef41182ba9ceb61c) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.

**Comment by dluetger** [comment]
2680 doesn't have text areas, but this is working now for 4192.
